<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitsWidget</class>
 <widget class="QWidget" name="CommitsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commits</string>
  </property>
  <property name="windowIcon">
   <iconset theme="git-commit">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="TreeView" name="treeViewRepo">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditFilter">
         <property name="placeholderText">
          <string>Filter...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="TreeView" name="treeViewHistory">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="LogDetailsWidget" name="textBrowser">
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>core/treeview.h</header>
  </customwidget>
  <customwidget>
   <class>LogDetailsWidget</class>
   <extends>QTextBrowser</extends>
   <header>widgets/logdetailswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
